### ISSUE TITLE
PDF of rules does not display rules with condition flags

### ISSUE THREAD

**Issue by itaibn** [TheoWinterhalter/formal-type-theory#42]
The PDF file `rule.pdf` advertised that it contains all the rules as if all the flags were on. However, as far as I can tell it actually doesn't display any of the rules that require a condition flag. For example, in `tt.v` there is a rule `TyProd` which follows `TySubst` and depends on the flag `withpi`, but this rule does not appear in the PDF.

**Comment by itaibn** [comment]
I tried fixing this issue by myself. The root of the problem appears to be the regex used to recognize rules (lines 198-202 of `coq2latex.py`):

```
            r'^\s*\|\s+'                   # the beginning of a rule
            r'(?P<rulename>\w+)\s*:\s*$'   # rule name
            r'\s*rule\s*'                  # header
            r'(?P<rulebody>.*?)'           # rule body
            r'endrule',                    # footer
```

The line labelled "header" does not account for the possibility of flags. However, when I change that line I see another error message which suggests that the code has another bug which makes it unable to handle of the flagged rules.

**Comment by TheoWinterhalter** [comment]
I think @andrejbauer is working on a new version that would allow to parametrise what rules you want to show. Sorry for the inconvenience.

**Comment by andrejbauer** [comment]
I closed this in #43 but wrongly referenced it there, sorry.
